css-layouts/layout-basics/src/exercises/exercise3/floats.css

```diff
@@ -0,0 +1,59 @@
+/* 
+  📚 Learn CSS - Exercise 3: Floats 📚
+
+  ✅ Here you're going to test out your knowledge of floating content
+  ✅ and clearing it after
+
+  🚨  Remember, you should only be ADDING new lines of code to 
```

Comment:
Nitpick: Let's line up the text with the 🚨so all the text starts at the same position. Remove an extra space before this line and the next one. Make sure this is true here and elsewhere in this pull request.